x/ecocredit/module/module.go

```diff
@@ -38,16 +39,55 @@ import (
 	"github.com/regen-network/regen-ledger/x/ecocredit/simulation"
 )
 
+var (
+	_ module.AppModule           = &Module{}
+	_ module.AppModuleBasic      = Module{}
+	_ module.AppModuleSimulation = Module{}
+)
+
 type Module struct {
+	key           storetypes.StoreKey
 	paramSpace    paramtypes.Subspace
 	accountKeeper ecocredit.AccountKeeper
 	bankKeeper    ecocredit.BankKeeper
 	Keeper        server.Keeper
 	authority     sdk.AccAddress
 }
 
+func (a Module) InitGenesis(s sdk.Context, jsonCodec codec.JSONCodec, message json.RawMessage) []abci.ValidatorUpdate {
+	_, err := a.Keeper.InitGenesis(s, jsonCodec, message)
+	if err != nil {
+		panic(err)
+	}
+	// TODO(Tyler): the app fails without this....????
+	return []abci.ValidatorUpdate{{PubKey: crypto2.PublicKey{}, Power: 4}}
+}
```

Comment:
will have to look more into this.. the tests fail on instantiation because of the empty updates 😢

Reply:
The workaround works for me. https://github.com/regen-network/regen-ledger/pull/1357#discussion_r951759030